src/parser/Tokenizer.h

```diff
@@ -313,6 +373,20 @@ struct TurtleToken {
         case '\\':
           res.push_back('\\');
           break;
+        case 'u': {
+          AD_CHECK(pos + 5 <= literal.size());
```

Comment:
The `<=` looks odd to myself, check and comment this method again + write Unit tests for it,
since it is the core of this PR

Reply:
It was indeed a mistake, and is fixed now.